conda/recipes/ucxx/meta.yaml

```
@@ -163,9 +178,16 @@ outputs:
         - {{ pin_subpackage('libucxx', exact=True) }}
         - ucx
         - rmm =23.08
+        {% if cuda_major == "11" %}
+        - cudatoolkit
+        {% else %}
+        - cuda-cudart-dev
+        {% endif %}
+        - cuda-version {{ cuda_version }}
       run:
         - python * *_cpython
         - packaging
+        - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
```

To make sure the `libcudart` dependency is satisfied at `run` time, would suggest adding `cudatoolkit` here (especially since we are relaxing the version constraint on it above

```suggestion
        {% if cuda_major == "11" %}
        - cudatoolkit
        {% endif %}
        - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
```